packages/jobs/build.mts

```diff
@@ -0,0 +1,16 @@
+import { build, defaultBuildOptions } from '@redwoodjs/framework-tools'
+
+// CJS build
+await build({
+  buildOptions: {
+    ...defaultBuildOptions,
+  },
+})
+
+// ESM build
```

Comment:
@Josh-Walker-GM Can I leave it to you to set up dual build here? Including the three different tsconfig files we use to get proper TS support everywhere, publint, attw etc